zingo-memo/src/lib.rs

```diff
@@ -4,16 +4,23 @@ use zcash_address::unified::{Address, Container, Encoding, Receiver};
 use zcash_client_backend::address::UnifiedAddress;
 use zcash_encoding::{CompactSize, Vector};
 
-pub const MEMO_VERSION: usize = 0;
-
+/// A parsed memo. Currently there is only one version of this protocol,
+/// which is a list of UAs. The main use-case for this is to record the
+/// UAs sent from, as the blockchain only records the pool-specific receiver
```

Comment:
I think this "from" should be "to".

Reply:
This code does not belong to this PR. I accidentally merged `dev` instead of rebasing to it, so the code emerged here.